test/Project.toml

```diff
@@ -10,3 +10,4 @@ LibGit2 = "76f85450-5226-5b5a-8eaa-529ad045b433"
 MacroTools = "1914dd2f-81c6-5fcd-8719-6d5c9610ff09"
 PlotlyBase = "a03496cd-edff-5a9b-9e67-9cda94a718b5"
 Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
+SnoopCompile = "aa65fe97-06da-5843-b5b1-d5d13cad87d2"
```

Comment:
🐎

Reply:
@rpkyle I've been experimenting with precompilation. But there is not everything simple, so I decided to postpone this issue for now. All that's left of it is the package in tests :)